test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -1659,6 +1659,27 @@ class InheritedClassTest : System.Attribute
         }
     }
 
+    Context "Script parameter completion" {
+        BeforeAll {
+            Setup -File -Path 'ScriptParamTabCompletionTest.ps1' -Content @'
+#requires -Modules ThisModuleDoesNotExist
```

Comment:
Original issue says about RunAsAdministrator. 
```suggestion
#requires -RunAsAdministrator
```
Since we run the test as regular user the test will check the original scenario. Will it pass?

Reply:
Don't we run tests both with and without elevated permissions? But yes, we can change the test to RunAsAdministrator if you prefer.

Reply:
Better to have both tests if we take into account modules.